3ds Max/Max2Babylon/Exporter/BabylonExporter.GLTFExporter.Texture.cs

```diff
@@ -58,10 +58,12 @@ private GLTFTextureInfo ExportBitmapTexture(GLTF gltf, BabylonTexture babylonTex
 
         private GLTFTextureInfo ExportTexture(BabylonTexture babylonTexture, GLTF gltf)
         {
-            return ExportTexture(babylonTexture, gltf, null, () =>
-            {
-                var sourcePath = babylonTexture.originalPath;
+            return ExportTexture(babylonTexture, gltf, null, 
+                () => { return TryWriteImage(gltf, babylonTexture.originalPath, babylonTexture.name); });
```

Comment:
Why do we want to refactor this into another method?

Reply:
see above, like for the delegate